Update iot-hub-python-python-file-upload.md

zolvarga opened the pull request:
Update sample code to use the file content instead of the file path in upload_blob_async API

azurecla commented:
Hi __@pcgeek86__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

tfitzmac commented:
@pcgeek86 thanks for submitting this change. It looks good. The PowerShell command needs to be changed to Get-AzureRmProviderOperation (without Resource)

pcgeek86 commented:
@tfitzmac Tom, sorry about that oversight. I've fixed it.

Cheers,
Trevor Sullivan
Microsoft MVP: PowerShell

tfitzmac commented:
@davidwrede this pull request can be merged.
Thanks Trevor!

Commit:
Update iot-hub-python-python-file-upload.md

Update sample code to use the file content instead of the file path in upload_blob_async API
